Formula/p/pkl.rb

```diff
@@ -1,8 +1,8 @@
 class Pkl < Formula
   desc "CLI for the Pkl programming language"
   homepage "https://pkl-lang.org"
-  url "https://github.com/apple/pkl/archive/refs/tags/0.27.2.tar.gz"
-  sha256 "804ab2dcd90c018da2976b7ecb118c069b9487b9800edc9cb198446661d9f685"
+  url "https://github.com/apple/pkl/archive/refs/tags/0.28.0.tar.gz"
+  sha256 "91a85f35e8d7bdffbca44dcc035cde33c8be8b4965ff869cfd810cc0f17b64e7"
```

Comment:
We re-did this tag; should be `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` now

Reply:
Oh, my bad! I forgot to pass the `-L` flag to curl 🤦

Reply:
no worries. :)